### PR TITLE
Added Y2 triumphs command

### PR DESCRIPTION
**Pull request by danburbul**

### CODE REVIEW

File: plugins/destiny.py [hunk 1]
```diff
@@ -907,6 +907,33 @@ def chars(text, nick, bot, notice):
         ))
     return "\x02{0}\x02: {1}".format(target['nick'], ' ; '.join(output))
 
+@hook.command('triumphs')
+def triumphs(text,nick,bot):
+    Y2_MOT_HASH = { '1872531696':'Challenge of the Elders','1872531697':'The Play\'s the Thing','1872531698':'The Third Element','1872531699':'This is Amazing','1872531700':'Eris Morn\'s Revenge','1872531701':'A Blade Reborn','1872531702':'Return to the Reef','1872531703': 'The Sword Logic'}
+    recordHashes = ['1872531696','1872531697','1872531698','1872531699','1872531700','1872531701','1872531702','1872531703']
```
Comment (JacobCallahan) [inline plugins/destiny.py]: no need for this. you can iterate through the Y2_MOT_HASH in the same way.

File: plugins/destiny.py [hunk 2]
```diff
@@ -907,6 +907,33 @@ def chars(text, nick, bot, notice):
...
+    membership = get_user(nick)
```
Comment (JacobCallahan) [inline plugins/destiny.py]: put this in an `else` below the following if statement.

File: plugins/destiny.py [hunk 3]
```diff
@@ -907,6 +907,33 @@ def chars(text, nick, bot, notice):
...
+            return 'When using gamertag you must also supply platform'
+        if platform in ['psn','ps','playstation','ps4']: platform = 2
+        if platform in ['xb1','xb','xbl','xbox']: platform = 1
+        membership = get_user(' '.join(text.split(' ')[0:len(text.split(' '))-1]),platform)
+    if type(membership) == str: return membership
+    for platform in [1,2]:
+        if platform in membership:
+            missing = []
+            if platform == 1: output.append('\x02(Xbox):\x02')
+            if platform == 2: output.append('\x02(Playstation):\x02')
```
Comment (JacobCallahan) [inline plugins/destiny.py]: you can change these two lines into one.
`output.append(CONSOLES[platform - 1])`

File: plugins/destiny.py [hunk 4]
```diff
@@ -907,6 +907,33 @@ def chars(text, nick, bot, notice):
...
+            book = get('{}{}/Account/{}/Advisors/?definitions=true'.format(BASE_URL,platform,membership[platform]['membershipId']),headers=HEADERS).json()['Response']['data']['recordBooks']['2175864601']
+            for hash in recordHashes:
```
Comment (JacobCallahan) [inline plugins/destiny.py]: s/recordHashes/Y2_MOT_HASH

**Comment by weylin** [comment]
Tested, LGTM.

**Pull request merged**